src/app/core/components/community-guidelines/community-guidelines.component.scss

```diff
@@ -1,5 +1,17 @@
-.page-container {
-  display: flex;
-  width: 75%;
-  flex-direction: column;
+@import '../../../../assets/scss/initializer';
+
+
+.terms-container{
+  h1 {
+    font-size: 52px;
+    font-weight: 700;
+    color: $color__share-section-heading;
+  }
```

Comment:
Did you test those styles on mobile aswell? 
Seems like there are missing media queries.

Reply:
You're right. Fixed it and added a little padding on mobile.